### PR TITLE
ZEPPELIN-748 make websocket maxTextMessageSize configurable

### PR DESCRIPTION
**Pull request by r-kamath**
### What is this PR for?
Allow the user to modify the value for  websocket maxTextMessageSize via conf. Recently a user filed an issue asking for the size to be configurable.

### What type of PR is it?
Improvement

### Todos
* [ ] - Task

### What is the Jira issue?
[ZEPPELIN-748](https://issues.apache.org/jira/browse/ZEPPELIN-748)

### How should this be tested?
modify the value for maxTextMessageSize in zeppelin-site.xml or zeppelin-env.sh

### Screenshots (if appropriate)
screen shot 2016-03-18 at 3 35 19 pm

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no 
* Does this needs documentation? yes. updated.

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Looks good to me

File: conf/zeppelin-site.xml.template [hunk 1]
```diff
@@ -219,5 +219,11 @@
   <description>Anonymous user allowed by default</description>
 </property>
 
+<property>
+  <name>zeppelin.websocket.maxTextMessageSize</name>
```
Comment (felixcheung) [inline conf/zeppelin-site.xml.template]: nit: should this be `zeppelin.websocket.max.text.message.size` for naming consistency?

**Comment by prabhjyotsingh** [comment]
Tested, LGTM.

File: conf/zeppelin-site.xml.template (hunk 1 above)
Comment (r-kamath) [inline conf/zeppelin-site.xml.template]: @felixcheung updated

File: conf/zeppelin-site.xml.template [hunk 2]
```diff
@@ -219,5 +219,11 @@
   <description>Anonymous user allowed by default</description>
 </property>
 
+<property>
+  <name>zeppelin.websocket.max.text.message.size</name>
+  <value>1024000</value>
+  <description>Size in characters of the maximum text message to be received by websocket. Defaults to 1024000</description>
```
Comment (felixcheung) [inline conf/zeppelin-site.xml.template]: btw, is this sort of the standard way to explain this? Is "character" as in byte?

Comment (r-kamath) [inline conf/zeppelin-site.xml.template]: @felixcheung that message is taken from http://archive.eclipse.org/jetty/8.0.0.RC0/apidocs/org/eclipse/jetty/websocket/WebSocket.Connection.html

Comment (felixcheung) [inline conf/zeppelin-site.xml.template]: ok thanks, that's what I thought.

**Comment by felixcheung** [comment]
merging if no more comment

**Pull request closed without merging**